pyproject.toml

```diff
@@ -20,6 +21,7 @@ python-jose = {extras = ["cryptography"], version = "^3.3.0"}
 [tool.poetry.group.dev.dependencies]
 geneweaver-testing = "^0.0.3"
 pytest-asyncio = "^0.21.0"
+magicmock =  "^0.03"
```

Comment:
Curious about this dependency. Just haven't used it before. Where do you use it in your tests?

Reply:
Thank you!. Removed this dependency. It is not used.